.gitignore

```diff
@@ -7,3 +7,5 @@ build/_output/*
 bin/*
 bundle/*
 cover.out
+data/crd/ssp.kubevirt.io_ssps.yaml
+data/olm-catalog/ssp-operator.clusterserviceversion.yaml
```

Comment:
I was mistaken, these files should not be ignored. But they don't need to be committed in this PR, they will be added in #44.